src/mixins/canvas_events.mixin.js

```diff
@@ -639,6 +656,14 @@
       (shouldRender || shouldGroup) && this.requestRenderAll();
     },
 
+    /**
+     * @private
+     * @param {Event} event Event object
+     */
+    _getPointerId: function(event) {
+      return this.enablePointerEvents ? event.pointerId : 1;
+    },
```

Comment:
what is the equivalent version on touch  events for determine wich touch in multi touch are we using? any idea?
i would prefer something like:
```js
return event.pointerId || 1;
```

unless pointerId can be 0.

Reply:
Fixed!

For touch events, `event.changedTouches[0].identifier` is the equivalent of `event.pointerId`.  For mouse events, no secondary pointers are relevant, so I use `0` as fallback value.